frontend/apps/cloudstorage/ftp.lua

```diff
@@ -26,14 +26,20 @@ function Ftp:downloadFile(item, address, user, pass, path, close)
         local file = io.open(path, "w")
         file:write(response)
         file:close()
-        UIManager:show(ConfirmBox:new{
-            text = T(_("File saved to:\n %1\nWould you like to read the downloaded book now?"),
-                path),
-            ok_callback = function()
-                close()
-                ReaderUI:showReader(path)
-            end
-        })
+        if G_reader_settings:readSetting("show_unsupported") then
```

Comment:
```suggestion
        if G_reader_settings:isTrue("show_unsupported") then
```

(Huh, this occurs quite a few times. :-P)